src/components/data/test/DataTable-test.js

```diff
@@ -434,26 +436,54 @@ describe('DataTable', () => {
       },
       // Clicking the same column when initially descending, transitions to ascending
       {
-        initialOrder: { field: 'name', direction: 'descending' },
+        startingOrder: { field: 'name', direction: 'descending' },
         clickedColumn: 'name',
         expectedNewOrder: { field: 'name', direction: 'ascending' },
       },
       // Clicking another column sets direction as ascending
       {
-        initialOrder: { field: 'name', direction: 'ascending' },
+        startingOrder: { field: 'name', direction: 'ascending' },
         clickedColumn: 'consistency',
         expectedNewOrder: {
           field: 'consistency',
           direction: 'ascending',
         },
       },
-    ].forEach(({ initialOrder, clickedColumn, expectedNewOrder }) => {
+      // Clicking another column sets direction default order when defined
+      {
+        startingOrder: { field: 'name', direction: 'ascending' },
+        clickedColumn: 'color',
+        expectedNewOrder: {
+          field: 'color',
+          direction: 'descending',
+        },
+      },
+      // No order sets direction as ascending
```

Comment:
```suggestion
      // Change sort column to a column with no initial order specified
```

Same comment as above. The difference here is that there is no initial order, but I think stating that in a comment would just repeat what's obvious from the example data.